uwhvz/settings/common.py

```diff
@@ -157,6 +157,11 @@
     messages.ERROR: 'danger',
 }
 
+REST_FRAMEWORK = {
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.PageNumberPagination',
+    'PAGE_SIZE': 10
```

Comment:
Doesn't this seem small for a `PAGE_SIZE`? Considering we can be returning things like a player list of above a 100 players, aggregating across 10 pages seems more work for no reason. Maybe increase this to something like 50?

Reply:
We're not though - we're returning one set of data for one player - 10 should be more than enough.